psnet/switch/switch.py

```diff
@@ -673,7 +678,36 @@ def auto_configure(self):
             module_logger.warn('{:} remains on the wrong '\
                                'subnet'.format(unmoveable))
 
+    def write_memory(self):
+        """
+        Save the current config to the switch so that it persists after next reboot.
+        """
+        # This is a privileged command: do we need/have the enable password?
+        if not self._enablepw and self._surveyer().check_mode(self.name):
+            self.get_enablepw()
 
+        commands = ['write memory']
+        cmd = self._surveyer()._cmd_runner(self._user,
+                                           self._pw,
+                                           self._enablepw,
+                                           self._port,
+                                           commands,
+                                           timeout=self.timeout,
+                                           priv=True)
+        try:
+            out_code,resp = cmd.run(self.name)
+        except IOError:
+            module_logger.info("Bad enable password!")
+            self._enablepw = None
+            out_code = 1
+            resp="Bad enable password"
+        except Exception:
+            out_code = 1
+            resp=""
```

Comment:
Not for you to fix, but the rest of the codebase doesn't utilize these `resp`, instead just dropping them on the floor

Reply:
I noticed this too and frowned at it